docs/variables.md

```diff
@@ -184,6 +184,18 @@ Only available for `successCondition`
 | `response.body` | Response body (`string`) |
 | `response.headers` | Response headers (`map[string][]string`) |
 
+### CronWorkflow Templates
+
+> v3.6 and after, note that the the variables here can be used with the full power of expr-lang expression.
+
+| Variable | Description|
+|----------|------------|
+| `cronworkflows.name` | Name of the CronWorkflow method (`string`) |
+| `cronworkflows.numActive` | Number of running CronWorkflows (`int`) |
+| `cronworkflows.lastScheduledTimeNull` | When `cronworkflows.lastScheduledTime` is a sane value this is set to `false` else `true` (`bool`) |
+| `cronworkflows.lastScheduledTime` | The time since this workflow was last scheduled (`time.Time`) |
+| `cronworkflows.phase` | Phase is an enum of `Active` or `Stopped`. It changes to `Stopped` when `stopStrategy.condition` is true (`string`) |
```

Comment:
Similarly I'm not sure what the value of exposing this is. It will always be "Active". It won't be evaluated when it is "Stopped".

Reply:
removed